client/webserver/site/src/html/bodybuilder.tmpl

```diff
@@ -90,4 +90,3 @@
 </body>
 </html>
 {{end}}
-5
```

Comment:
For reference and other reviewers, the reasoning:
- https://unix.stackexchange.com/a/18789/133297
- https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
- https://thoughtbot.com/blog/no-newline-at-end-of-file
- https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
is the "5" intentional?

Reply:
WTF.  No that has to be an accident.  I was just referring to the newline at the end of the file, but that "5" makes no sense there.